src/main/java/gregtech/api/metatileentity/multiblock/MultiblockControllerBase.java

```diff
@@ -381,6 +482,35 @@ public boolean onRightClick(EntityPlayer playerIn, EnumHand hand, EnumFacing fac
         return false;
     }
 
+    @Override
+    public boolean onWrenchClick(EntityPlayer playerIn, EnumHand hand, EnumFacing wrenchSide, CuboidRayTraceResult hitResult) {
+        if (!needsSneakToRotate() || playerIn.isSneaking()) {
+            if (wrenchSide == getFrontFacing() && allowsExtendedFacing()) {
+                if (!getWorld().isRemote) {
+                    setUpwardsFacing(upwardsFacing.rotateAround(EnumFacing.Axis.Y));
+                }
+                return true;
+            }
+        }
+        return super.onWrenchClick(playerIn, hand, wrenchSide, hitResult);
+    }
+
+    @Override
+    public boolean isValidFrontFacing(EnumFacing facing) {
+        return allowsExtendedFacing() || super.isValidFrontFacing(facing);
+    }
+
+    // todo tooltip on multis saying if this is enabled or disabled?
```

Comment:
I think a tooltip for the disabled state is a good idea. Maybe put in the tool usages section?